addons/payment_razorpay/models/payment_transaction.py

```
@@ -52,16 +51,12 @@ def _get_specific_rendering_values(self, processing_values):
         return_url_params = {'reference': self.reference}
 
         phone = self.partner_phone
-        error_message = _("The phone number is missing.")
         if phone:
             # sanitize partner phone
-            country_code = self.partner_country_id.code
-            country_phone_code = self.partner_country_id.phone_code
-            phone_info = phone_sanitize_numbers([phone], country_code, country_phone_code)
-            phone = phone_info[self.partner_phone]['sanitized']
-            error_message = phone_info[self.partner_phone]['msg']
-        if not phone:
-            raise ValidationError("Razorpay: " + error_message)
```

This should still raise when the phone number is not set.